frontend/apps/reader/modules/readerfooter.lua

```diff
@@ -272,11 +287,11 @@ function ReaderFooter:setupTouchZones()
 end
 
 -- call this method whenever the screen size changes
-function ReaderFooter:resetLayout()
+function ReaderFooter:resetLayout(force_reset)
     local new_screen_width = Screen:getWidth()
     local new_screen_height = Screen:getHeight()
     if new_screen_width == self._saved_screen_width
-        and new_screen_height == self._saved_screen_height then return end
+        and new_screen_height == self._saved_screen_height and force_reset == nil then return end
```

Comment:
Or perhaps `and not force_reset` so that nil and false are treated the same? (Unless the distinction between nil and false is important.)